src/evict/evict_lru.c

```diff
@@ -1852,7 +1852,7 @@ __evict_walk_file(WT_SESSION_IMPL *session,
 		mod = page->modify;
 		if (modified && txn_global->current != txn_global->oldest_id &&
 		    (mod->last_eviction_id == __wt_txn_oldest_id(session) ||
-		    !__wt_txn_visible_all(session, mod->update_txn)))
+		    !__wt_txn_visible_all(session, mod->update_txn, NULL)))
```

Comment:
Hmm, the `NULL` here is usually a sign that I've missed a case.  I suspect there's something to fix here.